tests/TestCase/Database/Driver/PostgresTest.php

```diff
@@ -173,4 +173,38 @@ public function testInsertReturning()
         $query = $translator($query);
         $this->assertSame('FOO', $query->clause('epilog'));
     }
+
+    /**
+     * Test that having queries replace the aggregated alias field.
+     *
+     * @return void
+     */
+    public function testHavingReplacesAlias()
+    {
+        $driver = $this->getMockBuilder('Cake\Database\Driver\Sqlserver')
```

Comment:
This test should be in the Driver/SqlserverTest.php file instead. That is likely why you're getting undefined constant errors.

Reply:
Thanks for the hint, the mocked driver was wrong indeed. But the constant errors were there because the mocked functions were not prepended with underscore anymore.